src/lib/bootloader/bootloader_factory.rb

```diff
@@ -59,8 +59,9 @@ def supported_names
           system_bl = nil
         end
         ret = system_bl ? [system.name] : [] # use current as first
-        ret << "grub2" unless Yast::Arch.aarch64 # grub2 everywhere except aarch64
-        ret << "grub2-efi" if Yast::Arch.x86_64 || Yast::Arch.aarch64
+        # grub2 everywhere except aarch64
```

Comment:
please update also comment here that risc is also efi only.

Reply:
good point, done!